docs/new-client.md

```diff
@@ -0,0 +1,76 @@
+# Writing a new editor client
+
+Before writing a new editor client, first check if someone else has managed to
+integrate Metals with your favorite text editor.
+
+* [VS Code](https://github.com/scalameta/metals/blob/master/vscode-extension/src/extension.ts),
+  maintained in this repo
+* [Atom](https://atom.io/packages/ide-scala), maintained by
```

Comment:
I think it's better if it link to the Github repo: https://github.com/laughedelic/atom-ide-scala